ProductMatrix/files/default.css

```diff
@@ -1,10 +1,12 @@
 div.productmatrix {
+	width:200px;
```

Comment:
That makes the div look different than others on the *ProductMatrix/product_view* page.